bin/gwdetchar-scattering

```diff
@@ -263,7 +263,7 @@ for i, seg in enumerate(statea):
                  obs=args.ifo[0], verbose=msg, nproc=args.nproc).resample(128))
 
 scatter_segments = DataQualityDict()
-
+a = []
```

Comment:
@siddharth101, can you call this variable something more descriptive?

```suggestion

actives = []
```

All the references to it below will also need to be updated, on lines 351 and 503.